testsuite/integration/iiop/src/test/config/arq/arquillian.xml

```diff
@@ -26,14 +26,14 @@
         <container qualifier="iiop-server" default="false">
             <configuration>
                 <property name="jbossHome">${basedir}/target/jbossas-iiop-server</property>
-                <property name="javaVmArguments">${server.jvm.args}</property>
+                <property name="javaVmArguments">${server.jvm2.args}  -Djboss.socket.binding.port-offset=100</property>
```

Comment:
It's "implementation detail", we do not depend on this property name. So safe to rename.